views/panel.js

```diff
@@ -55,6 +54,7 @@ Flame.Panel = Flame.View.extend({
         }.property('parentView.headerView'),
 
         labelView: Flame.LabelView.extend({
+            classNames: ['flame-panel-title'],
```

Comment:
Oh yeah... it creates this white space around the label view.

![screen shot 2015-09-15 at 15 52 41](https://cloud.githubusercontent.com/assets/2409276/9876885/044b6352-5bc2-11e5-9713-824fad2190d6.png)

If we change labelViews layout to: ```layout: { left: 0, right: 0, top: 0 },``` we get rid of that extra space. What do you think? Or is this completely wrong approach?